### DISCUSSION TITLE
Disable link-preview?

### DISCUSSION THREAD

**Discussion by mikro2nd** [zim-desktop-wiki/zim-desktop-wiki#1975] [Q&A] [answered]
Is there a way to disable the link-preview feature? By "link-preview" I mean the little balloon-tip that appears in the lower-left corner of a page when the cursor hovers over a link saying, "Go to `link`".

This seems to be a fairly recent feature, but, while others may like it, I find it very irritating, frequently obscuring text on the page, right where much of the page-editing action happens. A preference setting to disable it would be a happiness! :)

**Comment by introt** [comment, marked as answer]
No, there isn't currently an option in the GUI to disable the link overlay labels. I've opened PR #1980 to add it.

**Comment by ZtzEAcRsND** [comment]
@introt Would it be possible to float the pop-up near the link instead of at the bottom? e.g.,  holding the cursor over an Index entry displays path under where the cursor is hovering unless the Index entry is at the bottom of the page, then the pop-up is displayed above the hovered-over entry.  The same hover functionality for pathbar buttons, backlinks, tasklist entries, etc.

 Displaying the URL as a pop-up in the text window seems to have started after the bottom page status bar was replaced with the bottom edit-bar option.

**Comment by jaap-karssenberg** [comment]
What you are describing is a "tooltip" - we removed tooltips on links and
replaced it by the box in the lower left corner because multiple users
found the tooltip to much interfering, so the idea of the box at the bottom
was to get it out of the way. So bringing it back to tooltip will not fix
the problem and honestly, this is to much detailed UX to start adding
options where to put the box exactly.

-- Jaap

**Reply by mikro2nd** [reply]
Is there any reason not to merge the PR (#1908) which is just a simple "disable" option? - not fiddling around with detailed UI (which I agree would be reasons to pull hair out:)

**Comment by ZtzEAcRsND** [comment]
I didn't appreciate the reason for the change and agree more complexity isn't good. Thanks.
